service/src/test/java/skills/intTests/UserPointsSpecs.groovy

```diff
@@ -226,6 +226,28 @@ class UserPointsSpecs extends DefaultIntSpec {
         results2.data.get(0).totalPoints == 35
     }
 
+    def 'get project users with paging and minimum points'() {
+        when:
+        def results1 = skillsService.getProjectUsers(projId, 10, 1, "userId", true, "", 0)
```

Comment:
create user id via `getRandomUsers` utility (declared in test's parent) otherwise pki tests will fail